tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -198,12 +199,18 @@ Future<void> _main(List<String> args) async {
       var verbose = results.command!['verbose'];
       var watch = results.command!['watch'];
 
+      var config = GeneratorConfig.load();
```

Comment:
I think we'll need to be able to optionally pass in a directory to the load method, if that isn't a thing already.

Reply:
This can be a separate PR

Reply:
Is this enough?
https://github.com/serverpod/serverpod/blob/9423ecdbbb6b3308677efffe7d441fa1977640d5/tools/serverpod_cli/lib/src/generator/config.dart#L37
I think even an absolute path could be put into there.